stdlib/private/SwiftPrivateLibcExtras/SwiftPrivateLibcExtras.swift

```diff
@@ -81,6 +81,12 @@ public struct _stdlib_fd_set {
   }
 }
 
+#if arch(arm) || arch(i386)
+public typealias _stdlib_fd_set = __stdlib_fd_set<UInt32>
+#else
```

Comment:
Please list 64-bit platforms explicitly.  This ensures that this code breaks when we have a new port, and it is added to the appropriate list.